### ISSUE TITLE
Save State [Question]

### ISSUE THREAD

**Issue by agieocean** [ktock/container2wasm#234]
Very interested in this project but wondering if there's anything similar to v86's emulator.save_state functionality: [ex](https://github.com/copy/v86/blob/master/examples/save_restore.html) so there can be some level of persistence?

**Comment by ktock** [comment]
SGTM. [Bochs seems to support saving/restoring simulation](https://bochs.sourceforge.io/doc/docbook/user/using-save-restore.html) so we can use that feature.

**Comment by Cinemacloud** [comment]
Yes this is quite useful/important how difficult is it to implement the Bochs option in c2w sessions?
